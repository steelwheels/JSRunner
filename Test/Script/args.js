console.log("*** Test of Process.arguments\n") ;
console.log("args = " + Process.arguments + "\n") ;

