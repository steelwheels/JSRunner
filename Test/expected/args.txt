*** Test of Process.arguments
args = a,b,c
